README.md

```diff
@@ -5,6 +5,16 @@ Using python to make snake. How appropriate
 
 To view the package dependencies, see `./requirements.txt`.
 
+This game is based off the [tutorial](https://www.digitaljunky.io/make-a-snake-game-for-android-written-in-python-part-2/) by Alexis Matelin.
+
+
+## How to run the game
```

Comment:
You can leave a comment like this by pressing the **+** on the line number.

Reply:
@chilihedron @hboi  Try commenting on the code. See if there is anything you think needs addressing!